README.md

```diff
@@ -1 +1,311 @@
+URL хаба можно найти в логе запущенного `selenium-standalone` в строчке вида: `15:18:16.753 INFO - RemoteWebDriver instances should connect to: http://127.0.0.1:4444/wd/hub`.
+
+>**Важно!** Если у вас отсутствует соединение с сетью, нужно отключить туннель до `gemini-gui.haze.yandex.net`: в файле `.hermione.conf.js` выключить `(js)plugins['hermione-tunnel'].enabled = false`. ???
+
+<a name="report"></a>
+## Отчеты
+
+Результы выполнения тестов можно просмотреть:
+
+* В консоли (локально).
+* В конфигурации TeamCity (найти можно по номеру PR): 
+   * с помощью `teamcity reporter`, отчет доступен на вкладке `Tests`.
+
+![Картинка](file:hermione-report.jpg)
+
+   * с помощью `allure reporter`, oтчет доступен на вкладке `Hermione Allure Report`.
+
+![Изображение](https://jing.yandex-team.ru/files/gavryushin/SI%20%3A%3A%20SERP%20%3A%3A%20web4%20%3A%3A%20Pull%20requests%20%3A%3A%202.%20Hermione%20%3A%3A%20desktop%20>%20%234560%20%2803%20Mar%2016%2017%3A10%29%20>%20Overview%20—%20TeamCity%202016-03-03%2017-15-08.png)
+
+<a name="steps"></a>
+## Этапы разработки тестов
+ 
+ - Разработчик, тестировщик именеджер совместно продумывают и состовляют спецификацию для тестируемой функциональности, для каждой платформы. Затем, сохраняют ее способом принятым на сервисе. Например, в файле репозитория, рядом с кодом или в системе хранения тестовых сценариев.
+В СЕРПе сценарии хранятся в системе [TestPalm](https://testpalm.yandex-team.ru/serp) в секции «Steps» (см.[пример](https://testpalm.yandex-team.ru/testcase/serp-2458)).
+- Менеджер и тестировщик описывает сценарии по принципу «Что дано/Что сделали/Что Получили».
+[Пример сценария](https://st.yandex-team.ru/SERP-39936#1455118360000)
+- Разработчик пишет код и реализует тестовые сценарии. 
+- Тестировщик проверяет код тестов.
+	
+<a name="migration"></a>
+### Миграция тестов с Java на JavaScript
+
+- Разработчик создает задачу в ST для переработки автотеста на Java и переносит в нее  `id` теста  и описание сценария (секции «Steps») из [TestPalm](https://testpalm.yandex-team.ru/serp).
+- Определяет какие шаги в тесте может проверить с помощью `hermione`-тестов, а что с помощью `gemini`-тестов (визуальную составляющую).
+- В комментариях задачи переписывает часть спецификации теста, которая будет тестироваться с промощью `hermione`, в терминах приближенных к тестовым сценариям в виде цепочки шагов «действие -> ожидаемый результат» [см. пример](https://st.yandex-team.ru/SERP-39936#1455118360000)
+- Кодирует описанные действия c использованием `Mocha` и `Webdriver.io+Chai`
+- Тестировщик проверяет код созданных `hermione-` и `gemini-`тестов на полноту и правильность выполнения.
+
+<a name="create-test"></a>
+### Cоздание тестового сецнария
+
+Для разработки и запуска тестов используются следующие библиотеки:
+
+- [Mocha](https://mochajs.org) — содержит общие функции для тестирования, включая `describe` и `it`.
+- [WebdriverIO](http://webdriver.io/) — содержит универсальный API для управления поведением браузеров. 
```

Comment:
WebdriverIO is an open source testing utility for nodejs. It makes it possible to write super easy selenium tests with Javascript in your favorite BDD or TDD test framework. Even Cucumber tests are supported.
It basically sends requests to a Selenium server via the WebDriver Wire Protocol and handles its response. These requests are wrapped in useful commands, which provide callbacks to test several aspects of your site in an automated way.